certified-connectors/Progressus Advanced Projects/apiDefinition.swagger.json

```diff
@@ -0,0 +1,173 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Progressus Advanced Projects",
+    "description": "Use Progessus Advanced Projects connector to make odata calls to Progresssus Advanced Projects data in Power Automate or other tools.",
```

Comment:
@ProgressusGlobal The value 'description' contains at least one of the restricted words: 'power apps, power automate, logic apps'. 

Can you please make sure that you are following all the instructions in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)